spec/rubocop/comment_config_spec.rb

```diff
@@ -178,5 +181,10 @@ def disabled_lines_of_cop(cop)
       expect(disabled_lines_of_cop('Layout/LeadingCommentSpace'))
         .to eq([7, 8, 9, 50])
     end
+
+    it 'supports disabling cops for next line' do
+      expect(disabled_lines_of_cop('Style/DoubleNegation'))
+        .to eq([7, 8, 9, 56])
```

Comment:
The `disable all` from above threw me at first here. I was expecting to only see `56` at first.